kafka-consumer-spring/src/main/java/com/example/kafkademo/ConsumerExample.java

```diff
@@ -0,0 +1,13 @@
+package com.example.kafkademo;
```

Comment:
Hi @andreaTP 
Is there a way to club the classes/beans defined in the subpackages in one single file?

Reply:
Have you tried to place them in the body of the `SpringBootApplication` ?

Reply:
This is what I am getting for that approach:
```
Error creating bean with name 'consumerExample': Requested bean is currently in creation: Is there an unresolvable circular reference?
```

Reply:
We can dig into this and eventually solve it ... but keeping separated files seems more "idiomatic" for Spring users.
As you prefer 🙂